api/v1beta1/azuremanagedcontrolplane_webhook.go

```diff
@@ -772,21 +772,21 @@ func (m *AzureManagedControlPlane) validateAADProfileUpdateAndLocalAccounts(old
 		if m.Spec.AADProfile == nil {
 			allErrs = append(allErrs,
 				field.Invalid(
-					field.NewPath("Spec", "AADProfile"),
+					field.NewPath("spec", "aadProfile"),
 					m.Spec.AADProfile,
 					"field cannot be nil, cannot disable AADProfile"))
 		} else {
 			if !m.Spec.AADProfile.Managed && old.Spec.AADProfile.Managed {
 				allErrs = append(allErrs,
 					field.Invalid(
-						field.NewPath("Spec", "AADProfile.Managed"),
+						field.NewPath("spec", "aadProfile.managed"),
 						m.Spec.AADProfile.Managed,
 						"cannot set AADProfile.Managed to false"))
 			}
 			if len(m.Spec.AADProfile.AdminGroupObjectIDs) == 0 {
 				allErrs = append(allErrs,
 					field.Invalid(
-						field.NewPath("Spec", "AADProfile.AdminGroupObjectIDs"),
+						field.NewPath("spec", "aadProfile.AdminGroupObjectIDs"),
```

Comment:
```suggestion
						field.NewPath("spec", "aadProfile", "AdminGroupObjectIDs"),
```

Reply:
@willie-yao instead of `AdminGroupObjectIDs` it should be `adminGroupObjectIDs` camel case, Right?

Reply:
Yes that's correct, sorry for missing that!